maquetado/books-list.html

```diff
@@ -0,0 +1,58 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>Books list</title>
+  <meta name="description" content="books-list">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet"  type="text/css" href="application.css">
+</head>
+<body class="books-list-background">
+  <nav class="navbar">
+    <img src ="./assets/wolox.png" class="navbar-logo" alt="Wolox">
+  </nav>
+  <div class="books-list row">
+    <div class="book-preview column">
+      <img src ="./assets/book-cover.png" class="book-small-preview" alt="book cover">
+      <h2 class="bold">Titulo del libro</h2>
+      <h3 class="detail-color">Autor del libro</h3>
+    </div>
+    <div class="book-preview column">
+      <img src ="./assets/book-cover.png" class="book-small-preview" alt="book cover">
+      <h2 class="bold">Titulo del libro</h2>
+      <h3 class="detail-color">Autor del libro</h3>
+    </div>
+    <div class="book-preview column">
+      <img src ="./assets/book-cover.png" class="book-small-preview" alt="book cover">
+      <h2 class="bold">Titulo del libro</h2>
+      <h3 class="detail-color">Autor del libro</h3>
+    </div>
+    <div class="book-preview column">
+      <img src ="./assets/book-cover.png" class="book-small-preview" alt="book cover">
+      <h2 class="bold">Titulo del libro</h2>
+      <h3 class="detail-color">Autor del libro</h3>
+    </div>
+    <div class="book-preview column">
+      <img src ="./assets/book-cover.png" class="book-small-preview" alt="book cover">
+      <h2 class="bold">Titulo del libro</h2>
+      <h3 class="detail-color">Autor del libro</h3>
+    </div>
+    <div class="book-preview column">
+      <img src ="./assets/book-cover.png" class="book-small-preview" alt="book cover">
+      <h2 class="bold">Titulo del libro</h2>
+      <h3 class="detail-color">Autor del libro</h3>
+    </div>
+    <div class="book-preview column">
+      <img src ="./assets/book-cover.png" class="book-small-preview" alt="book cover">
+      <h2 class="bold">Titulo del libro</h2>
+      <h3 class="detail-color">Autor del libro</h3>
+    </div>
+    <div class="book-preview column">
+      <img src ="./assets/book-cover.png" class="book-small-preview" alt="book cover">
+      <h2 class="bold">Titulo del libro</h2>
+      <h3 class="detail-color">Autor del libro</h3>
+    </div>
+  </div>
+</body>
```

Comment:
This is already in a previous PR, if I'm not mistaken. Remember to chain them so we can see only the current changes, then you can change the base to master afterwards